persistence/src/main/java/org/cbioportal/genome_nexus/persistence/SignalMutationRepository.java

```diff
@@ -9,4 +9,11 @@
 {
     List<SignalMutation> findByHugoGeneSymbol(String hugoGeneSymbol);
     List<SignalMutation> findByHugoGeneSymbolIn(List<String> hugoGeneSymbol);
+    List<SignalMutation> findByChromosomeAndStartPositionAndEndPositionAndReferenceAlleleAndVariantAllele(
```

Comment:
@onursumer this name is a little funky but I guess it gets the point across :). Maybe something like getByGenomicLocationFields or Properties?

Reply:
@inodb the name has to be exactly like this to make it work automagically :) Spring Boot translates this into a proper DB query.